src/Components/navbar/navbar.css

```diff
@@ -1,45 +1,43 @@
-.navbar{
-    width: 100%;
-    height: 70px;
-    background-color: seagreen;
-    position: fixed;
-    top: 0;
-    float: left;
-    padding: 0;
-    margin: 0 0 3em 0;
-    list-style: none;
-    overflow: hidden;
+.navbar {
+  width: 100%;
+  height: 70px;
+  background-color: lightblue;
```

Comment:
I was just dickin around here when I realized there wasn't a formatter. Feel free to revert this however. I'm gonna try and get your "Contact Me" button to pop!